server/grpc.go

```diff
@@ -2,11 +2,13 @@ package server
 
 import (
 	"context"
-	pb "github.com/Hiroshii8/GoTrans/grpc/proto"
-	translate "github.com/Hiroshii8/GoTrans/translate"
-	"google.golang.org/grpc"
 	"log"
 	"net"
+
+	"google.golang.org/grpc"
+
```

Comment:
I think you can remove unnecesary newline in this import for more clean code @linehk